src/components/AddressInputPanel/index.tsx

```diff
@@ -23,7 +23,7 @@ const ContainerRow = styled.div<{ error: boolean }>`
   border-radius: 1.25rem;
   border: 1px solid ${({ error, theme }) => (error ? theme.red1 : theme.bg2)};
   transition: border-color 300ms ${({ error }) => (error ? 'step-end' : 'step-start')},
-    color 500ms ${({ error }) => (error ? 'step-end' : 'step-start')};
+  color: 500ms ${({ error }) => (error ? 'step-end' : 'step-start')};
```

Comment:
I thin this was intentional. color is part of `transition`, not its own property.

Reply:
fixed, nice catch!